hadoop-hdds/server-scm/pom.xml

```diff
@@ -165,6 +165,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.ozone</groupId>
+          <artifactId>hdds-test-utils</artifactId>
+          <scope>compile</scope>
```

Comment:
`hdds-test-utils` is for tests, should not be added in `compile` scope.

This seems to be unnecessary, compiles fine without this.